arc/conformance/plugin/arc_conformance.sh

```diff
@@ -186,6 +218,11 @@ if [ -z "${ARC_CLUSTER_RG}" ]; then
   python3 /arc/setup_failure_handler.py
 fi
 
+if [ -z "${OBJECT_ID}" ]; then
```

Comment:
What object id is this? Could you prefix the variable name to make it more descriptive?

Reply:
I have added comment for it.